From 582f6c945b7ab0d35db00bdc4c31cfb84f15f8e4 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Thu, 3 Nov 2022 19:50:45 +0000
Subject: [PATCH] app/meson.build: Add missing dep

Since we include the base private headers, we need the common base
sources to be generated.
---
 app/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/app/meson.build b/app/meson.build
index 14b20f6217..30665b6b36 100644
--- a/app/meson.build
+++ b/app/meson.build
@@ -37,6 +37,7 @@ libflatpak_app = static_library(
   dependencies : base_deps + [
     appstream_dep,
     json_glib_dep,
+    libflatpak_common_base_dep,
     libglnx_dep,
     libostree_dep,
     libsystemd_dep,
